html/wifi-networks.html

```diff
@@ -0,0 +1,96 @@
+              <strong><span id="errorSpan">{{.Error}}</span></strong> 
+            </div>
+            <div class="container pl-0">
+                <h2>WIfi Networks<br></h2>
+            </div>
+            <form method="POST">
+               <div class="col-md-12" >
+                    <h5 class="card-title"><strong>Add a Network</strong></h5>
+                    <div class="form-group">
+                        <label for="text-ssid">Network Name (SSID)</label>
+                        <div class="input-group">
+                          <input type="text" class="form-control" id="text-ssid" name="ssid" value="">
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="text-password">Password</label>
+                        <div class="input-group">
+                          <input type="password" class="form-control" id="text-password" name="password"  value="">
+                        </div>
+                    </div>
+                    <button type="submit" class="btn btn-primary">Add Network</button>
+                </div>
+            </form>
+
+            <br>
+
+            <h2>Current Networks<br></h2>
+            <div class="row">
+                 {{range .Networks}}
+                <div class="col-md-5 wifi-item" >
+                    <div class="card mb-3">
+                        <div class="card-body">
+                            <h5 class="card-title"><strong>{{.Ssid}}</strong></h5>
+                            {{if ne .Ssid "bushnet"}}
+                                <div class="text-right">
+                                    <a onclick="removeNetwork(this,{{.NetworkID}})" class="btn btn-info btn-md">
+                                      Remove 
+                                    </a>
+                                </div>
+                            {{end}}
+                        </div>
+                    </div>
+                </div>
+                {{end}}         
+            </div>
+        </div>
+
+
+        <script src="/static/js/jquery-3.3.1.slim.min.js"></script>
+        <script src="/static/js/bootstrap.min.js"></script>
+        <script>
+            function closeAlert(){
+                $("#errorAlert").css("visibility","hidden");
+            }
+            function removeNetwork(domLink, id){
+                fetch(
+                    "/wifi-networks/" + id,
+                    {method: "DELETE"}
+                ).then(function(response) {
+                   return response.json();
+                }).then(function(result) {
+                    if(result.length==0){
+                        $($(domLink).closest(".wifi-item")).remove()
+                    }else{
+                        $("#errorSpan").text(result);
+                        $("#errorAlert").css("visibility","visible");
+                    }
+                });
+            }
```

Comment:
The approach of just updating the UI for the bits the user has changed is nice but it does mean that if another user is making changes at the same time, the UI can get out of sync with the actual config.

I guess for this use case that's actually quite unlikely.